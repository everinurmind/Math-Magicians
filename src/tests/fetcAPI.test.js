import renderer from 'react-test-renderer';
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import Quote from '../components/FetchApi';

test('should render Quote.js', () => {
  const text = renderer.create(
    <BrowserRouter>
      <Routes>
        <Route path="/quote" element={<Quote />} />
      </Routes>
    </BrowserRouter>,
  );

  expect(text.toJSON()).toMatchSnapshot();
});
