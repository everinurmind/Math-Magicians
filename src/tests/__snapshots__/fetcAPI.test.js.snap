// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Quote.js 1`] = `null`;
